manifests/e/ETHZürich/SafeExamBrowser/3.3.2.413/ETHZürich.SafeExamBrowser.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ETHZürich.SafeExamBrowser

```

```suggestion
PackageIdentifier: ETHZurich.SafeExamBrowser
```
Windows and some CLI tools can be a bit finicky to work with special characters.

You'll need to update the publisher folder name from `ETHZürich` to `ETHZurich`